plugins/modules/cloud/digital_ocean/digital_ocean_droplet.py

```diff
@@ -54,6 +54,11 @@
      - add an additional, private network interface to droplet for inter-droplet communication.
     default: False
     type: bool
+  vpc_uuid:
+    description:
+     - A string specifying the UUID of the VPC to which the Droplet will be assigned. If excluded, Droplet will be
+       assigned to the account's default VPC for the region.
+    type: str
```

Comment:
```suggestion
    type: str
    version_added: 0.2.0
```
If the PR can get merged the next days, this is needed. Otherwise it must be `version_added: 1.0.0` (in the hope the PR gets merged until then).